js/deer-config.js

```diff
@@ -192,16 +191,20 @@ export default {
                             fetch(`${tiny}overwrite`, {
                                 method: "PUT",
                                 mode: 'cors',
+                                headers: {
+                                    "Content-Type": "application/json",
+                                    "Authorization": `Bearer ${DLA_USER.authorization}`
+                                },
                                 body: JSON.stringify(list_project)
                             }).then(r => r.ok ? r.json() : Promise.reject(Error(r.text)))
-                                .catch(err => alert(`Failed to save: ${err}`))
+                            .catch(err => alert(`Failed to save: ${err}`))
 
-                            fetch(`${tiny}overwrite`, {
-                                method: "PUT",
-                                mode: 'cors',
-                                body: JSON.stringify(list_public)
-                            }).then(r => r.ok ? r.json() : Promise.reject(Error(r.text)))
-                                .catch(err => alert(`Failed to save: ${err}`))
+                            // fetch(`${tiny}overwrite`, {
```

Comment:
what is this tra$h?

Reply:
Ah it should be uncommented.  It was in the original code and manage-mss.html was firing both of them.  I wasn't sure what was going on and didn't want that overwrite to keep firing at the public list so I commented it out at some point.